src/components/molecules/ReviewCard.jsx

```diff
@@ -23,17 +24,23 @@ const ReviewCard = ({ selectedCategory, categoryParam }) => {
       {reviews.map((review) => {
         console.log(review.review_img_url, "<<<imageURL");
         return (
-          <div key={review.review_id} className={styles.card}>
-            <TextField text={`Title: ${review.title}`} />
-            <img
-              src={review.review_img_url}
-              alt={`Image for review with title of: ${review.title}`}
-            ></img>
-            <TextField text={`Author: ${review.owner}`} />
-            <TextField text={`Category: ${review.category}`} />
```

Comment:
I would prefer your text to have more structure than just being <TextField> elements that are all <p> tags - it reduces accessibility in my opinion when H1/h2 and more semantic tags could be really useful. Everything being a <p> tag at its core and named the same <TextField> elements is not a clear structure